corehq/apps/domain/calculations.py

```diff
@@ -347,6 +348,22 @@ def all_domain_stats():
     }
 
 
+def domain_metrics(domain_obj, id, all_stats):
+    props = calced_props(domain_obj, id, all_stats)
+    metrics_dict = {
+        metrics_attr: props.get(props_key)
+        for metrics_attr, props_key in DOMAIN_METRICS_TO_PROPERTIES_MAP.items()
+    }
+    metrics_dict['domain'] = domain_obj.name
+    metrics_dict['last_modified'] = datetime.now(tz=timezone.utc)
+
+    # these are properties on the Django model, so don't try to write them
```

Comment:
nit: could be a bit more explicit to say these are "calculated fields on the Django model" or something like that. I recognize that it is a tough situation since we have referred to the DomainMetrics as calculated properties, but hopefully going forward we can limit where we use "calculated properties" to avoid conflicting with the conventional definition of that term.

Reply:
What would you think of "these are model properties on DomainMetrics"? "Calculated fields" gets the point across just as well but it doesn't feel as technically correct to me.

Reply:
Oh interesting. In my mind, calculated (or computed) is technically correct when referring to these properties/fields of a model class, since they aren't stored in the table but instead calculated/computed after fetching the object from the database.

Reply:
That's fair actually, I guess it maybe depends whether you're looking at it from the database's perspective or the code's -- whether it's a "calculated" field versus a regular field, or if it's a "property" versus a regular model method.

Reply:
https://github.com/dimagi/commcare-hq/pull/35522/commits/54d4affeb95843996d43fb4dec1d33e259d67dc0